scripts/examples/gn_silabs_example.sh

```diff
@@ -300,7 +300,7 @@ else
     if [ "$SILABS_BOARD" == "BRD4338A" ]; then
         echo "Compiling for 917 WiFi SOC"
         USE_WIFI=true
-        optArgs+="chip_device_platform =\"SiWx917\" is_debug=false "
+        optArgs+="chip_device_platform =\"SiWx917\" is_debug=false sl_uart_log_output=true "
```

Comment:
`sl_uart_log_output` is declared in silabs_board.gni. Set it to true directly in that file so it doesn't have to depend on this build script...

Reply:
updated the silabs_board.gni